src/BehaviorsSDKManaged/ManagedUnitTests/ManagedUnitTests.nuget.props

```diff
@@ -0,0 +1,29 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Condition=" '$(ExcludeRestorePackageImports)' != 'true' ">
+    <RestoreSuccess Condition=" '$(RestoreSuccess)' == '' ">True</RestoreSuccess>
+    <RestoreTool Condition=" '$(RestoreTool)' == '' ">NuGet</RestoreTool>
+    <ProjectAssetsFile Condition=" '$(ProjectAssetsFile)' == '' ">$(MSBuildThisFileDirectory)project.lock.json</ProjectAssetsFile>
+    <NuGetPackageRoot Condition=" '$(NuGetPackageRoot)' == '' ">$(UserProfile)\.nuget\packages\</NuGetPackageRoot>
+    <NuGetPackageFolders Condition=" '$(NuGetPackageFolders)' == '' ">C:\Users\DAVAUGH\.nuget\packages\;C:\Microsoft\Xamarin\NuGet\</NuGetPackageFolders>
```

Comment:
>C:\Users\DAVAUGH [](start = 70, length = 16)

I guess we all have to rename our user accounts now :-)

BTW, since we didn't have these props and targets before I have to ask why do we need them now?

Reply:
The .gitignore file was missing the NuGet v3 file types. I've added those in and removed the nuget.props files from the PR.